src/bsk_rl/sats/satellite.py

```diff
@@ -229,7 +230,7 @@ def action_description(self) -> Any:
         """Human-interpretable description of action space."""
         return self.action_builder.action_description
 
-    def is_alive(self, log_failure=False) -> bool:
+    def is_alive(self, log_failure=True) -> bool:
```

Comment:
This is set to true because lines 241-242 short circuit and ensure that logging will only happen once